### PR TITLE
FIX: Fix memory issue with some gatk4 modules

### PR DESCRIPTION
**Pull request by maxulysse**
Plus minimal code polish

### CODE REVIEW

**Review by asp8200 — APPROVED** [review]

File: modules/nf-core/gatk4/createsequencedictionary/main.nf [hunk 1]
```diff
@@ -20,7 +20,7 @@ process GATK4_CREATESEQUENCEDICTIONARY {
     script:
     def args = task.ext.args ?: ''
 
-    def avail_mem = 6
+    def avail_mem = 6144
```
Comment (FriederikeHanssen) [inline modules/nf-core/gatk4/createsequencedictionary/main.nf:23]: Why so much for such a small process

**Review by FriederikeHanssen — APPROVED** [review]

File: modules/nf-core/gatk4/createsequencedictionary/main.nf (hunk 1 above)
Comment (asp8200) [reply modules/nf-core/gatk4/createsequencedictionary/main.nf:23]: `avail_mem` was accidentally decreased from 6GB to 6MB which - as you may already know - caused some problems over in Sarek. I guess the purpose of this PR is just to get things working again; not to find the lowest still-working value of `avail_mem`.

**Review by pontus — APPROVED** [review]

**Review by matthdsm — APPROVED** [review]

File: modules/nf-core/gatk4/createsequencedictionary/main.nf (hunk 1 above)
Comment (FriederikeHanssen) [reply modules/nf-core/gatk4/createsequencedictionary/main.nf:23]: yes well aware of the problem, just questioning why has 6GB by default assigned in comparison to all others. two birds one stone and so on :)

Comment (matthdsm) [reply modules/nf-core/gatk4/createsequencedictionary/main.nf:23]: The picard equivalent only has 3GB. Also, and I've said this before, why do we have duplicate modules?

**Review by nvnieuwk — APPROVED** [review]

**Comment by maxulysse** [comment]
The failed linting left will be fixed by https://github.com/nf-core/modules/pull/3499

**Comment by maxulysse** [comment]
and same for the failing test

**Pull request merged**